consensus/ibft/ibft.go

```diff
@@ -364,6 +513,8 @@ func (i *Ibft) runSyncState() {
 			return isValidator
 		})
 
+		updateSnapshotCallback(oldLatestNumber)
+
```

Comment:
`oldLatestNumber` doesn't change after bulk sync is done. Is this ok?
(Let's think we start sync from 10 and could do bulk sync to 40 and synced one by one to 48. In this case, first `updateSnapshotCallback` is called with 10 and second should be called with 40? You don't need to update oldLatestNumber?)

Reply:
You're right 👍 

It would be updated only in the next loop.

I've also noticed that we can jump out of the loop quicker if we updated the snapshot store on a per block basis with PoS.

Fixed 👍 

d24f46e